reservation_units/models.py

```diff
@@ -238,6 +238,47 @@ class ReservationUnit(models.Model):
         on_delete=models.PROTECT,
     )
 
+    PRICE_UNIT_PER_15_MINS = "per_15_mins"
+    PRICE_UNIT_PER_30_MINS = "per_30_mins"
+    PRICE_UNIT_PER_HOUR = "per_hour"
+    PRICE_UNIT_PER_HALF_DAY = "per_half_day"
+    PRICE_UNIT_PER_DAY = "per_day"
+    PRICE_UNIT_PER_WEEK = "per_week"
+    PRICE_UNIT_FIXED = "fixed"
+    PRICE_UNITS = (
+        (PRICE_UNIT_PER_15_MINS, _("per 15 minutes")),
+        (PRICE_UNIT_PER_30_MINS, _("per 30 minutes")),
+        (PRICE_UNIT_PER_HOUR, _("per hour")),
+        (PRICE_UNIT_PER_HALF_DAY, _("per half a day")),
+        (PRICE_UNIT_PER_DAY, _("per day")),
+        (PRICE_UNIT_PER_WEEK, _("per week")),
+        (PRICE_UNIT_FIXED, _("fixed")),
+    )
+    price_unit = models.CharField(
```

Comment:
Would be nice to have some `help_text` to tell what these prices mean. It would also be rendered into gql api when these fields gets publishe to there, but as well for us good info.

Reply:
Good point, fixed. 🙂